### PR TITLE
Set modification time after closing file. Fixes #82

### LINKED ISSUE #82: Downloaded file has incorrect modification time
On Windows, a newly-downloaded file (created via `skicka download /Remote C:\Local`) has its modification time set to "now", rather than the modification time of the remote file.

Re-running `skicka download` syncs the metadata and corrects the local modification time.

The file should have the correct (remote) modification time after its initial download.

I suspect this occurs because [`defer writeCloser.Close()`](https://github.com/google/skicka/blob/5f49b1882556ed62cc217fa94c124349ed5e252c/download.go#L339) resets the modification time (to now) after `os.Chtimes` runs.

### PR DESCRIPTION
**Pull request by bgrainger**

### CODE REVIEW

**Comment by bgrainger** [comment]
This will call `os.Chtimes` even if other functions have failed.

File: download.go [hunk 1]
```diff
@@ -357,7 +359,7 @@ func downloadFile(f *gdrive.File, localPath string, progressBar *pb.ProgressBar)
 
 	verbose.Printf("Downloaded and wrote %d bytes to %s", f.FileSize, localPath)
 
-	return os.Chtimes(localPath, normalizeModTime(f.ModTime), normalizeModTime(f.ModTime))
+	return err
```
Comment (mmp) [inline download.go]: Won't this return the 'err' from the "writeCloser, err := ..." line at the top of the function?  I think here you just want "return"?  (But even then, it's not completely to clear if the value of err returned from the function is in fact modify-able by the defer statement.

Instead, how about removing the "defer writeCloser.Close()" from the original function, manually closing it in the case where downloadDriveFile returns an error, and also closing it before the Chtimes call?

Comment (bgrainger) [inline download.go]: `err` is reassigned inside the defered func created on line 335. However, the logic is very convoluted (and incorrect if some other error occurred) so I already closed this PR.

**Comment by bgrainger** [comment]
Opened a new PR: #84

**Pull request closed without merging**